doc/source/raysgd/raysgd_ft.rst

```diff
@@ -0,0 +1,31 @@
+RaySGD Fault Tolerance
+======================
+
+.. note:: Fault tolerance is currently only enabled for the PyTorchTrainer.
+
+For distributed deep learning, jobs are often run on infrastructure where nodes can be pre-empted frequently (i.e., spot instances in the cloud). To overcome this, RaySGD provides **fault handling** features that enable training to continue regardless of node failures.
+
+.. code-block:: bash
+
+    trainer.train(max_retries=N)
+
+
+How does it work?
+-----------------
+
+During each ``train`` method, each parallel worker iterates through the dataset, synchronizing gradients and parameters at each batch. These synchronization primitives can hang when one or more of the parallel workers becomes unresponsive (i.e., when a node is lost). To address this, we've implemented the following protocol.
+
+  1. If any worker node is lost, Ray will mark the training task as complete (``ray.wait`` will return).
+  2. Ray will throw ``RayActorException`` when fetching the result for any worker, so we will call ``ray.get`` on the "finished" training task.
```

Comment:
The use of "we" here doesn't match the perspective of the rest of the doc

Reply:
replace with "The Trainer class" ?

Reply:
done